warrior/WarriorCore/Classes/junit_class.py

```diff
@@ -237,7 +237,10 @@ def _junit_to_html(self, junit_file, print_summary=True):
         """ Convert junit file to html"""
         if not hasattr(self, 'html_result_obj'):
             self.html_result_obj = WarriorHtmlResults(junit_file)
-        self.html_result_obj.generate_html(junit_file, None)
+        if print_summary is True:
+            self.html_result_obj.generate_html(junit_file, None, True)
```

Comment:
changed it, however i did it this way because i wasn't sure if print_summary would always be a bollen